lib/plugin/retryFailedStep.js

```diff
@@ -2,7 +2,13 @@ const event = require('../event');
 const recorder = require('../recorder');
 
 const defaultConfig = {
-  retries: 3,
+  /**
+     * Bug with retryFailedStep.
+     * It doesn't use our custom config and doesn't change amount of retryes
+     * https://github.com/codeceptjs/CodeceptJS/issues/3268
+     * https://github.com/codeceptjs/CodeceptJS/issues/2829
+     */
+  retries: 2,
```

Comment:
It will be good to add unit test on this. test/unit/plugin/retryFailedStep_test.js